### DISCUSSION TITLE
Could not load type 'MySql.Data.MySqlClient.MySqlConnection'

### DISCUSSION THREAD

**Discussion by luis3199correia** [OrchardCMS/OrchardCore#16117] [Q&A]
### Describe the bug
I'm using Pomelo.EntityFrameworkCore.MySql 3.1.1 and I get this error when I change my Orchard Core version 1.0.0-rc2-15532 to 1.0.0-rc2-15852.
`Could not load type 'MySql.Data.MySqlClient.MySqlConnectionStringBuilder' from assembly 'MySqlConnector, Version=1.0.0.0, Culture=neutral, PublicKeyToken=d33d3e53aa5f8c92'.`

### To Reproduce
When running the app this error appear
![image](https://user-images.githubusercontent.com/32116636/191805168-4e35d582-e4ea-47b9-8a90-e20c06d4560e.png)

**Comment by sebastienros** [comment]
Try to reference the `MySql.Data.MySqlClient` package directly from your csproj with a version that will work. However there might be some incompatibility between the version that yessql is using and the one Pomelo.EntityFrameworkCore.MySql is. Maybe we are using `MySqlConnectionStringBuilder` now that doesn't exist in one of the versions.

**Comment by luis3199correia** [comment]
Did you find any incompatibility?
